Python/Product/AnalysisBrowser/MultipleMemberView.cs

```diff
@@ -44,5 +46,38 @@ class MultipleMemberView : MemberView {
                 return _children;
             }
         }
+
+        public override void ExportToDiffable(
+            TextWriter writer,
+            string currentIndent,
+            string indent,
+            Stack<IAnalysisItemView> exportStack,
+            out IEnumerable<IAnalysisItemView> exportChildren
+        ) {
+            var children = new HashSet<string>();
+            foreach (var c in Children) {
+                string displayValue = "";
+                if (c is ClassView) {
```

Comment:
Considered that, but I'd still want the type filtering here. e.g. `ModuleRefView.DisplayValue` would have something valuable to display, but we'd still want to exclude it here for the purposes of the export.